pymc3/model.py

```diff
@@ -13,6 +13,60 @@
 __all__ = ['Model', 'Factor', 'compilef', 'fn', 'fastfn', 'modelcontext', 'Point', 'Deterministic', 'Potential']
 
 
+class InstanceMethod(object):
```

Comment:
Kind of annoying we have to jump through these hoops here but I suppose this is still the cleanest way... unless we depend on `dill` instead of `pickle`.